Presentation.Web/App_Start/SwaggerConfig.cs

```diff
@@ -135,7 +135,20 @@ public static void Register()
                     c.ResolveConflictingActions(apiDescriptions => apiDescriptions.First());
 
                     //Do not enable the build-in caching in the odata provider. It caches error responses which we dont want so we wrap it in a custom caching provider which bails out on errors
-                    c.CustomProvider(defaultProvider => new CustomCachingSwaggerProvider(new ODataSwaggerProvider(defaultProvider, c, GlobalConfiguration.Configuration)));
+                    ODataSwaggerProvider CreateOdataSwaggerProvider(ISwaggerProvider defaultProvider)
+                    {
+                        return new ODataSwaggerProvider(defaultProvider, c, GlobalConfiguration.Configuration)
+                            .Configure
+                                (
+                                    //without navigation properties enabled, the odata model's "value" property will be omitted from swagger output
+                                    //We then apply the OnlyIncludeReadModelSchemasInSwaggerDocumentFilter to ensure the page will still render.
+                                    //The entire odata model is huge because of the many circular references to large object types, so during dom update,
+                                    //the swagger ui just crashes even if the swagger json is valid
+                                    //also, the swagger odata provider does not respect that some properties have been removed from the edm model so we must remove them manually in the filter
+                                    configure => configure.IncludeNavigationProperties() 
```

Comment:
this is what actually enabled the models (the "value" property) that was missing when using `OdataListResponse<T>`